build.gradle

```diff
@@ -56,17 +56,17 @@ subprojects {
 
   // NOTE sourceCompatibility & targetCompatibility are set in gradle.properties to meet requirements of Gradle
   // Must redefine here to work around a bug in the Eclipse plugin
-  sourceCompatibility = targetCompatibility = JavaVersion.VERSION_1_8
+  sourceCompatibility = targetCompatibility = JavaVersion.VERSION_11
 
-  plugins.withType(JavaPlugin) {
-    project.tasks.withType(JavaCompile) { task ->
-      if (JavaVersion.current().isJava11Compatible()) {
-        task.options.release = 8
-      }
+  plugins.withType(JavaPlugin).configureEach {
```

Comment:
I wonder if it's necessary to keep this. `sourceCompatibility` & `targetCompatibility` are already set to 11 and the release pipeline uses Java 11.

Reply:
Good question.
I don't know either. From the comments it looks like the other statement was only a workaround for Eclipse.
Let me remove it.

Reply:
My rationale was simpler since 11 is the minimum there's no point in setting 8. And since the release runs on Java 11, there's no way a release can have a different binary version.
But the eclipse point is also good 👍

Reply:
You're totally right. And I think I removed the wrong part.
I should have left this I guess and change it to 11 (in case we would update the release build to use Java 17 compiling for Java 11):
```
  plugins.withType(JavaPlugin) {
    project.tasks.withType(JavaCompile) { task ->
      if (JavaVersion.current().isJava11Compatible()) {
        task.options.release = 8
      }
    }
    project.tasks.withType(GroovyCompile) { task ->
      if (JavaVersion.current().isJava11Compatible()) {
        task.options.release = 8
      }
    }
  }
```

Reply:
LGTM

Reply:
Do you think this is good to merge? 
I wouldn't want to merge as long as your review is still red 😁

Reply:
tbh, my mistake, I did the formal "review" out of work custom, I am not a fan of such formalities for this.

Reply:
I am happy if you find things and point me to them :)